docs/power_consumption.md

```diff
@@ -0,0 +1,67 @@
+# Power Consumption
+
+## Intro
+
+We decided to take a closer look at the occurrence of energy consumption on apu
+platforms. Measuring energy is valuable if we strive to improve firmware.
+Thanks to this information, we are also able to detect errors quickly.
+
+## Setup testing enviroment
+
+Measuring power consumption is possible thanks to the connection of RTE with the
```

Comment:
I don't understand "RTE".    I do understand about the INA219, and power measurement in general.   I think this is just "in the power cable, add an INA219 and some device that can be used to log data".     So is this a way to hook up the INA219 to the apuN so that it can log itself, or is it a separate logger?  Separate seems better, since part of the point is to observe the boot process.

(And thank you for providing this documentation -- I think attention to power consumption is important.)

Reply:
I explained a little more of "RTE", we receive logs from the sensor on RTE, which has a special image with added sensor support and thus it is possible.